docs/section21/Rule21-2.md

```diff
@@ -0,0 +1,62 @@
+
+# Boiler - Rule 21-2  
+
+**Rule ID:** 21-2  
+**Rule Description:** For purchased HW or steam loops in the proposed design, baseline should include all on-site distribution pumps included in the proposed design.  
+**Rule Assertion:** B-RMR LEN(ASHRAE229:conditioning_component.pump) = P-RMR LEN(ASHRAE229:conditioning_component.pump) for pumps serving purchased hot water or steam loops  
+**Appendix G Section:** Section 21 Boiler  
+**Appendix G Section Reference:** Section G3.1.1.3.4 Building System-Specific Modeling Requirements for the Baseline model  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  
```

Comment:
Applicability should be that project uses purchased heating based on Rule 21-1.

Reply:
This is not a mandatory RDS so suggest to not implement it. I am generally confused about the relevant App G requirements. G3.1.3.5 requires baseline heating to be modeled with primary only loop, but proposed design may have primary/secondary arrangement with multiple loops and pumps. G3.1.1.3.4 (which is the focus of this RDS) requires  modeling all on-site distribution pumps the same in both baseline and proposed design. So should proposed design be modeled based on the baseline requirements and not based on what's specified?